RadixWallet/Features/AppFeature/Overlay/InfoLinkSheet+Reducer.swift

```diff
@@ -63,6 +63,11 @@ extension InfoLinkSheet {
 		case securityshield
 		case buildsecurityshields
 		case nohardwaredevice
+		case device
```

Comment:
I would prefer nesting here instead of flat, e.g. `case infoAboutFactors(FactorsInfo)`.

Reply:
on a regular enum I would've gone with that categorization for sure. However, we are using the enum cases `rawValue` (which we wouldn't be able to have with nested cases) to support navigation from one glossary item to another. This is, if you are looking at a text that is provided like this:
```
is that users can “stake” [XRD tokens](?glossaryAnchor=xrd) to increase the security of the network
```

then we can link to the user to `InfoLinkSheet.GlossaryItem.xrd` when they tap on `XRD tokens`. It is definitely easier to maintain if we have it this way (`?glossaryAnchor=device`), than if we need to support something more complicated such as `?glossaryAnchor=infoaboutfactors.device`